sibylapp/view/utils/formatting.py

```diff
@@ -55,3 +58,13 @@ def format_two_contributions_to_view(
             axis="columns",
         )
     return compare_df
+
+
+def show_probability_select_box():
+    st.sidebar.checkbox(
+        "Show probability",
+        value=st.session_state["display_proba"],
+        help="Display prediction in terms of probability",
+        disabled=not config.SUPPORT_PROBABILITY,
```

Comment:
I think the UI would be less confusing if we just didn't have this checkbox at all when SUPPORT_PROBABILITY is disabled, since there is no point in that situation where it would become enabled